torch/_functorch/_aot_autograd/subclass_parametrization.py

```diff
@@ -14,7 +14,7 @@ def forward(self, *tensors) -> torch.Tensor:  # type: ignore[no-untyped-def]
             }
             todo = todo[:-num_children]
             rebuilt = tp.__tensor_unflatten__(d, meta, None, None)  # type: ignore[attr-defined]
-            todo.append(rebuilt)
+            todo.insert(0, rebuilt)
```

Comment:
?

Reply:
Let's say you have TwoTensor(TwoTensor_a, TwoTensor_b), then the way we put stuff into rebuilt queue looks like: [TwoTensor_b, TwoTensor_a, TwoTensor]. As a result, when we first build TwoTensor_b and if we put it in the end (before my PR), the original structure is lost as it would like sth like: [plain_tensor, plain_tensor, TwoTensor_b] and when we build TwoTensor_a, it will think plain_tensor and TwoTensor_b are the elements of TwoTensor_a